<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QListWidget" name="uploaderList"/>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="quickMode">
     <property name="text">
      <string>Quick mode (mouse release screenshots)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Delay before taking a screenshot</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="delay">
     <property name="toolTip">
      <string>In seconds</string>
     </property>
     <property name="whatsThis">
      <string>A delay before taking a screenshot, in seconds</string>
     </property>
     <property name="suffix">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="5">
    <widget class="QListWidget" name="hotkeys"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Hotkeys</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Encoder settings</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Still image format</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QComboBox" name="imageFormatBox"/>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Recording format</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QCheckBox" name="captureCursor">
     <property name="text">
      <string>Capture cursor</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="nameScheme">
     <property name="toolTip">
      <string>http://doc.qt.io/qt-5/qdatetime.html#toString</string>
     </property>
     <property name="text">
      <string>Screenshot %(yyyy-MM-dd HH:mm:ss)date.%ext</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>File name scheme:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QCheckBox" name="hideToTray">
     <property name="text">
      <string>Pressing &lt;X&gt; hides to tray</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QComboBox" name="formatBox">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="settingsButton">
     <property name="text">
      <string>Open settings directory</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Uploader selection:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
